src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/PedigreeAnnotation.java

```diff
@@ -67,21 +73,27 @@ private void initializeSampleDBAndSetFounders(GATKPath pedigreeFile) {
      */
     protected Set<Trio> getTrios() {
         if (pedigreeFile!= null) {
-            final SampleDBBuilder sampleDBBuilder = new SampleDBBuilder(PedigreeValidationType.STRICT);
-            sampleDBBuilder.addSamplesFromPedigreeFiles(Collections.singletonList(pedigreeFile));
-            return sampleDBBuilder.getFinalSampleDB().getTrios();
+            return getSampleDB().getTrios();
         }
         return Collections.emptySet();
     }
 
     /**
-     * Setter for pedigree file and founderIDs to be used by the GATKAnnotationPluginDescriptor to handle duplicated annotaiton
+     * Setter for pedigree file and founderIDs to be used by the GATKAnnotationPluginDescriptor to handle duplicated annotation
      * arguments between InbreedingCoeff and ExcessHet
      */
     public void setPedigreeFile(GATKPath pedigreeFile) {
         this.pedigreeFile = pedigreeFile;
         hasAddedPedigreeFounders = false;
     }
+
+    /**
+     * Setter for pedigreeValidationType
+     */
+    public void setPedigreeValidationType(PedigreeValidationType pedigreeValidationType) {
+        this.pedigreeValidationType = pedigreeValidationType;
+    }
```

Comment:
The existing setters are here only to accommodate the annotation descriptor, but this isn't integrated with it. Hopefully it can be removed if the constructor overload solves the problem.

Reply:
I considered trying to expose pedigreeValidationType to the plugin descriptor (which would be consistent with GATK3's behavior and ability to set that on the command line), but backed away from that b/c I assumed you would not like the more invasive change.